airflow/dag_processing/processor.py

```diff
@@ -375,21 +376,16 @@ def manage_slas(self, dag: DAG, session: Session = None) -> None:
             .group_by(TI.task_id)
```

Comment:
@ashb @uranusjr  i think we can add a `filter(TI.map_index==-1)` to exclude mapped tasks.  this, combined with a check when initializing a mapped operator  (check that no SLA  / warn) should effectively disable SLAs for mapped tasks i think....